nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/GetMongoIT.java

```diff
@@ -581,6 +581,47 @@ public void testDateFormat() throws Exception {
         Assert.assertTrue(format.matcher((String) result.get("date_field")).matches());
     }
 
+    @Test
+    public void testBadDatabaseNameRoutesToError() {
```

Comment:
I'm getting errors on all these added tests, is there something I have to set up in my integration test environment (perhaps a variable registry or something)?

Reply:
No. Weird thing is I just ran it and there were not errors.